README.md

```diff
@@ -1,92 +1,120 @@
 [![npm version](https://img.shields.io/npm/v/@google-cloud/translate.svg)](https://www.npmjs.org/package/@google-cloud/translate)
 [![codecov](https://img.shields.io/codecov/c/github/googleapis/nodejs-translate/master.svg?style=flat)](https://codecov.io/gh/googleapis/nodejs-translate)
 
-The [Cloud Translation API](https://cloud.google.com/translate/docs), can dynamically translate text between thousands of language pairs. The Cloud Translation API lets websites and programs integrate with the translation service programmatically. The Cloud Translation API is part of the larger Cloud Machine Learning API family.
 
 
-* [Using the client library](#using-the-client-library)
+
+Cloud Translation API Client Library for Node.js
+
+
+* [Cloud Translation Node.js Client API Reference][client-docs]
+* [Cloud Translation Documentation][product-docs]
+* [github.com/googleapis/nodejs-translate](https://github.com/googleapis/nodejs-translate)
+
+Read more about the client libraries for Cloud APIs, including the older
+Google APIs Client Libraries, in [Client Libraries Explained][explained].
+
+[explained]: https://cloud.google.com/apis/docs/client-libraries-explained
+
+**Table of contents:**
+
+
+* [Quickstart](#quickstart)
+  * [Before you begin](#before-you-begin)
+  * [Installing the client library](#installing-the-client-library)
+  * [Using the client library](#using-the-client-library)
 * [Samples](#samples)
 * [Versioning](#versioning)
 * [Contributing](#contributing)
 * [License](#license)
 
-## Using the client library
+## Quickstart
 
-1.  [Select or create a Cloud Platform project][projects].
+### Before you begin
 
+1.  [Select or create a Cloud Platform project][projects].
 1.  [Enable billing for your project][billing].
-
-1.  [Enable the Google Cloud Translation API API][enable_api].
-
+1.  [Enable the Cloud Translation API][enable_api].
 1.  [Set up authentication with a service account][auth] so you can access the
     API from your local workstation.
 
-1. Install the client library:
+### Installing the client library
+
+```bash
+npm install @google-cloud/translate
+```
 
-        npm install --save @google-cloud/translate
 
-1. Try an example:
+### Using the client library
 
 ```javascript
-// Imports the Google Cloud client library
-const {Translate} = require('@google-cloud/translate');
-
-// Your Google Cloud Platform project ID
-const projectId = 'YOUR_PROJECT_ID';
-
-// Instantiates a client
-const translate = new Translate({
-  projectId: projectId,
-});
-
-// The text to translate
-const text = 'Hello, world!';
-// The target language
-const target = 'ru';
-
-// Translates some text into Russian
-translate
-  .translate(text, target)
-  .then(results => {
-    const translation = results[0];
-
-    console.log(`Text: ${text}`);
-    console.log(`Translation: ${translation}`);
-  })
-  .catch(err => {
-    console.error('ERROR:', err);
-  });
+async function quickstart(
```

Comment:
It would be nice to update the quickstart to follow the `main` model described here:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head